### ISSUE TITLE
Determine Group Call participants (names)

### ISSUE THREAD

**Issue by PavelGladko** [Microsoft/BotBuilder-RealTimeMediaCalling#14]
Could you please clarify how it is possible to determine call participants (Names).
At the moment I look into HueBot example RealTimeMediaCall class.
To be more specific OnIncomingCallReceived() event with RealTimeMediaIncomingCallEvent parameter that have Conversation property with list of Participants. Each participant has internal identity property that is for example : "29:1c2fgntp......". 
Is there a way knowing that participant Identity obtain name of Skype account that takes part in a Call?
Maybe there is another way to determine List of Skype Accounts that take part in a Call?

**Comment by MalarGit** [comment]
Few things.. First bots in group calls is currently not supported.  Huebot demonstrates only 1:1 call. 

The identity in the Conversation is intentionally scrubbed to guard privacy. That 29:xxx uniquely identifies that participant. You can use that id to send messages to them. Is there any reason why you would need the actual skype account?

**Comment by kronwell** [comment]
Just a note to this. If you create standard text messaging bot, skype account name is during conversation available.

**Comment by MalarGit** [comment]
@kronwell  Yes you do get the name of the user. But I think Pavel is asking about the Skype account(xx@yy.com) of the user.

**Comment by kronwell** [comment]
@MalarGit I tried HueBot sample and property DisplayName of Participant object was null, only Identity was filled. But I had the same problem with IVR bot in times before real time calling SDK was available...

**Comment by MalarGit** [comment]
@kronwell  you are right. The display name comes only for text messaging and not for calling scenarios.

**Comment by kronwell** [comment]
@MalarGit Any chance to change it? ;-)

**Comment by MalarGit** [comment]
@kronwell will check and let you know.

**Comment by kronwell** [comment]
@MalarGit Thanks

**Comment by PavelGladko** [comment]
Actually Name of the User (DisplayName) would perfectly fit my needs. Thanks

**Comment by MalarGit** [comment]
@PavelGladko @kronwell  thanks for the feedback. This issue has been added to our backlog.
